src/Uno.PackageDiff/Program.cs

```diff
@@ -18,26 +18,77 @@
 
 namespace Uno.PackageDiff
 {
-    class Program
+	class Program
 	{
 		private static readonly PackageSource NuGetOrgSource = new PackageSource("https://api.nuget.org/v3/index.json");
 
 		static async Task<int> Main(string[] args)
-        {
+		{
 			string sourceArgument = null;
 			string targetArgument = null;
 			string outputFile = null;
 			string diffIgnoreFile = null;
+			string outputType = null;
+			string githubPAT = null;
+			string sourceRepository = null;
+			string githubPRid = null;
 
 			var p = new OptionSet() {
 				{ "base=", s => sourceArgument = s },
 				{ "other=", s => targetArgument = s },
 				{ "outfile=", s => outputFile = s },
 				{ "diffignore=", s => diffIgnoreFile = s },
+				{ "outtype=", s => outputType = s},
+
+				//
+				// GitHub PR comments related
+				//
+				{ "github-pat=", s => githubPAT = s },
+				{ "source-repository=", s => sourceRepository = s },
+				{ "github-pr-id=", s => githubPRid = s  }
 			};
 
 			p.Parse(args);
 
+			IDiffWriter writer = null;
+			if(string.IsNullOrEmpty(outputType))
+			{
+				writer = new Writers.MDDiffWriter(outputFile);
+			}
+			else
+			{
+				var types = outputType.Split(",", StringSplitOptions.RemoveEmptyEntries);
+				if(types.Length == 0)
+				{
+					writer = new Writers.MDDiffWriter(outputFile);
+				}
+				else
+				{
+					var composite = new Writers.CompositeWriter();
+					foreach(var type in types)
+					{
+						if(string.Equals("md", type, StringComparison.OrdinalIgnoreCase))
+						{
+							composite.Add(new Writers.MDDiffWriter(outputFile));
+						}
+						else if(string.Equals("diff", type, StringComparison.OrdinalIgnoreCase))
+						{
+							composite.Add(new Writers.XmlIngnoreWriter(outputFile));
+						}
+						else if(string.Equals("github", type, StringComparison.OrdinalIgnoreCase))
+						{
+							composite.Add(new Writers.GitHubWriter(githubPAT, sourceRepository, githubPRid));
+						}
+					}
+					if(composite.Count == 0)
+					{
+						Console.WriteLine($"Error : Invalid outtype {outputType}.");
+						return 1;
+					}
```

Comment:
a combination of valid and invalid `outtype` will slip through here... ex: "outtype=diff,asd" 

you should add a final else: `else { Console.WriteLine($"Error: Invalid outtype: {type}"); }`
and change the final if to `composite.Count != types.Length`